velodyne_pointcloud/src/conversions/transform.cc

```diff
@@ -43,38 +42,15 @@ namespace velodyne_pointcloud
       ROS_ERROR_STREAM("Could not load calibration file!");
     }
 
-    config_.target_frame = config_.fixed_frame = "velodyne";
-    tf_ptr_ = boost::make_shared<tf::TransformListener>();
-
-    if(config_.organize_cloud)
-    {
-      container_ptr = boost::shared_ptr<OrganizedCloudXYZIR>(
-          new OrganizedCloudXYZIR(config_.max_range, config_.min_range, config_.target_frame, config_.fixed_frame,
-                                  config_.num_lasers, data_->scansPerPacket(), tf_ptr_));
-    }
-    else
-    {
-      container_ptr = boost::shared_ptr<PointcloudXYZIR>(
-          new PointcloudXYZIR(config_.max_range, config_.min_range,
-                              config_.target_frame, config_.fixed_frame,
-                              data_->scansPerPacket(), tf_ptr_));
-    }
-
```

Comment:
It is not necessary (dublicate code) because the reconfigure callback is always executed with the params defined in the .launch file as soon a node starts.